addons/sale_product_configurator/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_sale_product_configurator","access.sale.product.configurator","model_sale_product_configurator","sales_team.group_sale_manager",1,0,1,0
```

Comment:
`sales_team.group_sale_salesman`

Reply:
are you sure? the salesman can not create products I think

Reply:
the product configurator is not used to create products but to select the correct variant based on attribute selection when adding a SO line, nothing to do with product creation despite the misleading name :p

Reply:
:mind_blow:

Reply:
It may create a product_product if needed with the attributes combination, but I think it's done in sudo :D.